### PR TITLE
New UI for presentation/video swap and fullscreen buttons

### PR DESCRIPTION
**Pull request by pedrobmarin**
Fixes #6507

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/styles.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+:root {
+  --color-transparent: #ff000000;
+  ::-webkit-media-controls {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/styles.scss]: -webkit-media-controls seems unrecoginzed. Where did you get this from? When I search for it I only see longer forms as in https://gist.github.com/afabbro/3759334

**Comment by antobinary** [comment]
Tested with locked webcam viewer (Lock settings) - seems some code was removed and we are now allowed to try to share even when webcams are locked - I see the video preview

**Comment by antobinary** [comment]
I still see 
screenshot from 2019-01-25 15-35-02
after clicking on fullscreen button (nothing breaks though - which is nice :) )

**Comment by pedrobmarin** [comment]
I'll take another look on this next week, then

File: bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/styles.scss (hunk 1 above)
Comment (capilkey) [reply bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/styles.scss]: It's accurate. https://css-tricks.com/custom-controls-in-html5-video-full-screen/

**Comment by pedrobmarin** [comment]
> I still see
> screenshot from 2019-01-25 15-35-02
> after clicking on fullscreen button (nothing breaks though - which is nice :) )

For some reason this never happens to me. Do you get this with any browser?

> Tested with locked webcam viewer (Lock settings) - seems some code was removed and we are now allowed to try to share even when webcams are locked - I see the video preview

I see the problem. What should be the expected behavior when webcams are locked?

**Comment by antobinary** [comment]
Merging this PR so we can start using it. However, Please take a look at relocating the fullscreen of webcam button (or audio state one) as they currently overlap
screenshot from 2019-01-28 12-31-58

**Pull request merged**